scripts/appbuilder

```diff
@@ -8,12 +8,10 @@
 function print_usage {
     echo "$(basename $0): a useful shell script for converting Docker images into Mystikos cpio archives"
     echo
-    echo "Usage: $(basename $0) [-o file] [-d [file]] [[-i] image] [-e extras] [-v]"
+    echo "Usage: $(basename $0) [-o file] [-i image] [-e extras] [-v] [[-d] file]"
```

Comment:
Some people don't like to be forced to place an argument at a particular location.

Reply:
the location matters only for the default situation(no other options like -o -i )